commec/utils/file_utils.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Module for FileTools, containing Static functions useful for dealing with common file parsing tasks.
+"""
+
+import argparse
+import os
+
+class FileTools():
+    """
+    Static function container, useful for dealing with common file parsing tasks.
+    """
+
+    # Can be moved somewhere else.
+    @staticmethod
+    def is_empty(filepath: str) -> bool:
+        """
+        is_empty
+
+        usage: check that a file is empty
+        input:
+        - name of file
+        """
+        try:
+            abspath = os.path.abspath(os.path.expanduser(filepath))
+            return os.path.getsize(abspath) == 0
+        except OSError:
+            # If there is an error (including FileNotFoundError) consider it empty
+            return True
```

Comment:
```suggestion
def is_empty(filepath: str) -> bool:
    """Check if a file is empty or non-existent."""
    try:
        return os.path.getsize(os.path.abspath(os.path.expanduser(filepath))) == 0
    except OSError:
        # Errors such as FileNotFoundError considered empty
        return True
```

Looking at this, we can simplify; I don't think we need that kind of usage info in the docsting (even though I think I might have written it earlier...)